cmd/config/internal/commands/cmdcreatesetter.go

```diff
@@ -20,14 +20,16 @@ import (
 func NewCreateSetterRunner(parent string) *CreateSetterRunner {
 	r := &CreateSetterRunner{}
 	set := &cobra.Command{
-		Use:     "create-setter DIR NAME VALUE",
-		Args:    cobra.ExactArgs(3),
+		Use:     "create-setter DIR NAME --value VALUE",
+		Args:    cobra.RangeArgs(2, 3),
 		Short:   commands.CreateSetterShort,
 		Long:    commands.CreateSetterLong,
 		Example: commands.CreateSetterExamples,
 		PreRunE: r.preRunE,
 		RunE:    r.runE,
 	}
+	set.Flags().StringVar(&r.Set.SetPartialField.Setter.Value, "value", "",
```

Comment:
alternative to specifying the value as an argument.  may be used to specify values that start with '-'

Reply:
Done.